_data/projects/sharkio.yml

```diff
@@ -0,0 +1,10 @@
+name: Sharkio
+desc: Sharkio is an API proxy for you development environments. It records all the requests that are sent to your API. It allows you to replay the requests easily. Generate types, documentation, test suites, And interact with your API using AI.
+site: https://github.com/idodav/sharkio
+tags:
+  - help wanted
+  - good first issue
+  - enhancements
```

Comment:
🤔 these tabs should be used to describe the project - what technologies does it use, what problem does it solve, etc.

These current labels are unnecessary and won't add much value once listed.